vhdbuilder/packer/install-dependencies.sh

```diff
@@ -153,20 +153,10 @@ fi
 INSTALLED_RUNC_VERSION=$(runc --version | head -n1 | sed 's/runc version //')
 echo "  - runc version ${INSTALLED_RUNC_VERSION}" >> ${VHD_LOGS_FILEPATH}
 
-## for ubuntu-based images, cache multiple versions of runc
 if [[ $OS == $UBUNTU_OS_NAME ]]; then
-  RUNC_VERSIONS="
-  1.0.0-rc92
-  1.0.0-rc95
-  1.0.3
-  "
-  if [[ $(isARM64) == 1 ]]; then
-    # RUNC versions of 1.0.3 later might not be available in Ubuntu AMD64/ARM64 repo at the same time
-    # so use different version set for different arch to avoid affecting each other during VHD build
-    RUNC_VERSIONS="
-    1.0.3
-    "
-  fi
+  
+  RUNC_VERSIONS="1.1.5+azure-ubuntu-${UBUNTU_RELEASE}u1_${CPU_ARCH}" 
```

Comment:
isn't this entirely unused because it's installed in standalone containerd ?

Reply:
it's used for downloading, and then installation is triggered through standalone containerd

Reply:
Good point. That's called before these downloads, so it should not be necessary. Removed it and got green builds again (in test all VHDs pipeline).